easycv/models/detection/utils/__init__.py

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env python3
 # Copyright (c) 2014-2021 Megvii Inc. All rights reserved.
-
-from .boxes import bboxes_iou, postprocess
-from .utils import output_postprocess
+from .boxes import (bbox2result, bbox_overlaps, bboxes_iou, box_cxcywh_to_xyxy,
```

Comment:
Check that all common apis under utils directory, and  move to common utils

Reply:
done